appveyor.yml

```diff
@@ -25,6 +24,17 @@ test:
     - '**\Sarif.Driver.UnitTests.dll'
     - '**\Sarif.FunctionalTests.dll'
     - '**\Sarif.ValidationTests.dll'
+    
+artifacts:
+  - path: 'bld\bin'
```

Comment:
I don't think we'll be able to use appveyor to produce official packages because we won't have the right access to sign them w/ Microsoft key.